README.md

```diff
@@ -65,3 +65,23 @@ podman-compose -f deploy/podman/observability.yaml up
 ```
 
 The Grafana and Prometheus web UIs are then accessible on `http://localhost:3000` and `http://localhost:9090`, respectively.
+
+
+## Public Service Enrollment
+
+A public version of FlightCtl is running on a OpenShift cluster managed by Red Hat. This provide the capability of having a public service for enrolling your own devices and showcase FlightCtl.
```

Comment:
That service is for internal development/demos, so not accessible publicly. How about we keep this section generic, describing the general process how to build and use a bootc image with the agent and configuring it for a given service?

Reply:
ack